Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -3,18 +3,33 @@
 import xbot.common.command.BaseCommand;
 import xbot.common.math.PIDFactory;
 import competition.subsystems.elevator.ElevatorSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+
 import com.google.inject.Inject;
 import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import competition.operator_interface.OperatorInterface;
 
 public class ElevatorMaintainerCommand extends BaseCommand {
 
+    OperatorInterface oi;
     ElevatorSubsystem elevator;
     PIDManager pid;
+    double startTime;
```

Comment:
Rename to `initializationTime`

Reply:
I removed this variable and just called the timer in initialize to get the giveUpCalibrationTime.